src/Symfony/Component/Console/Command/DumpCompletionCommand.php

```diff
@@ -55,7 +55,7 @@ protected function configure()
         $this
             ->setHelp(<<<EOH
 The <info>%command.name%</> command dumps the shell completion script required
-to use shell autocompletion (currently, bash and fish completion is supported).
+to use shell autocompletion (currently, bash, zsh and fish completion is supported).
```

Comment:
I'm not sure, but should we use `are` here?

```suggestion
to use shell autocompletion (currently, bash, zsh and fish completion are supported).
```

Reply:
indeed, we should